src/main/java/database/operations/CompanyRepository.java

```diff
@@ -0,0 +1,132 @@
+package database.operations;
+
+import database.data.Company;
+import database.data.JdbcConnectionUtil;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyRepository {
+
+    private static final String SELECT_ALL  = "SELECT * FROM companies";
+    private static final String SELECT_ID = "SELECT * FROM companies WHERE id = ?";
+    private static final String INSERT= "INSERT INTO companies(name, address) VALUES(?, ?)";
+    private static final String DELETE_ID = "DELETE FROM companies WHERE id = ?";
+    private static final String UPDATE_ID = "UPDATE companies SET name = ?, address = ? WHERE id = ?";
+
+    public List<Company> selectAll() throws SQLException {
+        Connection connection = null;
+        Statement statement = null;
+        ResultSet resultSet = null;
+        List <Company> reslist = new ArrayList <>();
+        try {
```

Comment:
You could simplify it using try-with-resources.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html